### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by danfoy**
Updated to match format of [X9Developers/block-explorer][1].

[1]: https://github.com/X9Developers/block-explorer/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,8 +1,32 @@
...
+## Other
+- `master` branch should never be broken, it contains the current version running in production.
+
+## Environment
+It is simpler to use the recommended developer environment:
+
+### infrastructure
+- See [infra/](https://github.com/danfoy/crypto-coin-alerts/tree/develop/infra)
+
+### server
```
Comment (AlexITC) [inline CONTRIBUTING.md]: the project name is alerts-server.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,8 +1,32 @@
...
+- See [alerts-server/](https://github.com/danfoy/crypto-coin-alerts/tree/develop/alerts-server)
 
+### ui
```
Comment (AlexITC) [inline CONTRIBUTING.md]: the project name is alerts-ui

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,8 +1,32 @@
 # Contributing
+This is an incomplete list of things to take care while contributing.
 
-This is an incomplete list of things to take care while contributing:
-- Every commit message should have this format `$project: title` being `$project` one of `alerts-server`, `alerts-ui` or `root`, see the [commit list](https://github.com/AlexITC/crypto-coin-alerts/commits/master) to get a better idea.
-- Try to keep the code style while we integrate a code formatter tool like scalafmt.
+## Commits
+Every commit message should have this format `$project: title` being `$project` one of `alerts-server`, `alerts-ui` or `infra`. See the [commit list](https://github.com/AlexITC/crypto-coin-alerts/commits/master) to get a better idea.
```
Comment (AlexITC) [inline CONTRIBUTING.md]: missing dash to keep the bullets.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,8 +1,32 @@
...
+
+- Stick to one topic per commit, and don't alter anything not referenced in the commit message.
```
Comment (AlexITC) [inline CONTRIBUTING.md]: please, use the same text.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,8 +1,32 @@
...
+## Code style
+- Try to keep the code style while we integrate a code formatter tool like scalafmt.
+
+## Pull requests
+- Pull requests should go to the `develop` branch.
+
+## Other
+- `master` branch should never be broken, it contains the current version running in production.
+
+## Environment
```
Comment (AlexITC) [inline CONTRIBUTING.md]: please, use the same content, just update the project names.

**Comment by AlexITC** [comment]
It might be worth looking at the raw source file, I would use this and just update names: https://raw.githubusercontent.com/X9Developers/block-explorer/develop/CONTRIBUTING.md

**Comment by danfoy** [comment]
No problem, will make the changes when I get home

**Comment by AlexITC** [comment]
That's totally fine, they disappear because the places where the comments where made doesn't exist anymore.

Thanks!

**Pull request merged**